portal/src/components/Materials/MaterialInfo/MaterialInfo.component.less

```diff
@@ -94,39 +94,19 @@
         }
       }
     }
-    &_part_of {
-      display: flex;
-      flex-direction: row;
-      margin-bottom: 30px;
 
-      &:before {
-        content: '';
-        position: relative;
-        background: #0077c8;
-        height: auto;
-        width: 4px;
-        border-radius: 2px;
-      }
-    }
-    &_main {
-      padding: 8px 16px;
+    &_collections {
+      padding: 0;
       display: flex;
-      flex-direction: column;
-      cursor: pointer;
+    }
 
-      h3 {
-        margin-bottom: 20px;
-        display: flex;
+    &_collection {
+      width: 50%;
+      &:first-child{
+        margin-right: 10px;
       }
-
-      &_arrow {
-        content: '';
-        display: inline-flex;
-        width: 18px;
-        height: 33px;
-        margin-left: 20px;
-        background: url('/images/arrow-bold.svg') 50% 50% no-repeat;
-        background-size: contain;
+      &:nth-child(n + 3) {
+        display: none;
```

Comment:
waarom is dit?

Reply:
dat zat er al in, op een andere plek. dat heb ik niet zelf verzonnen